File: src/main/java/sk/hudak/prco/dto/EshopProductInfoDto.java
package sk.hudak.prco.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.ToString;

@ToString
@Getter
@AllArgsConstructor
public class EshopProductInfoDto implements DtoAble {
    private long countOfAllProduct;
    private long countOfAlreadyUpdated;
}

File: src/main/java/sk/hudak/prco/dto/EshopProductInfoDto.kt
package sk.hudak.prco.dto;

public class EshopProductInfoDto implements DtoAble {

    private long countOfAllProduct;
    private long countOfAlreadyUpdated;

    public EshopProductInfoDto(long countOfAllProduct, long countOfAlreadyUpdated) {
        this.countOfAllProduct = countOfAllProduct;
        this.countOfAlreadyUpdated = countOfAlreadyUpdated;
    }

    @Override
    public String toString() {
        return "EshopProductInfoDto{" +
                "countOfAllProduct=" + countOfAllProduct +
                ", countOfAlreadyUpdated=" + countOfAlreadyUpdated +
                '}';
    }

    public long getCountOfAllProduct() {
        return countOfAllProduct;
    }

    public long getCountOfAlreadyUpdated() {
        return countOfAlreadyUpdated;
    }
}
